### PR TITLE
strings.xml

### PR DESCRIPTION
**Pull request by FantomasXX**
m1721: parts: Introduce Chinese language.

### CODE REVIEW

**Comment by tdrkDev** [comment]
@FantomasXX tomas rename commit to m1721: parts: Introduce Chinese language please

**Comment by rtx4d** [comment]
Looks good to me, after changing name for commit, we can merge this

**Comment by rtx4d** [comment]
@rtx4d

**Comment by FantomasXX** [comment]
Thank you for your guidance

**Comment by FantomasXX** [comment]
m1721: parts: Introduce Chinese language

**Review by FantomasXX — COMMENTED** [review]
1

File: parts/res/values-cn/strings.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ ​    <​string​ ​name​=​"​mback_summary​"​>mBack 震感反馈和触摸音量设置</​string​> 
+ ​    <​string​ ​name​=​"​mback_vibration_strength_title​"​>震感反馈设置</​string​> 
+ ​    <​string​ ​name​=​"​vibration_strength_list_title​"​>震感反馈强度</​string​> 
+ ​    <​string​ ​name​=​"​mback_touch_sound_volume_title​"​>触摸反馈音量</​string​> 
+ ​    <​string​ ​name​=​"​mback_volume_title​"​>音量设置</​string​> 
+ ​    <​string-array​ ​name​=​"​vibration_strength_entries​"​> 
+ ​        <​item​>关闭</​item​> 
+ ​        <​item​>轻微</​item​> 
+ ​        <​item​>适中</​item​> 
+ ​        <​item​>酥麻</​item​>
```
Comment (urain39) [inline parts/res/values-cn/strings.xml:35]: 字面翻译更好。
Literally translation is good enough.

File: parts/res/values-cn/strings.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ ​     distributed under the License is distributed on an "AS IS" BASIS, 
+ ​     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ ​     See the License for the specific language governing permissions and 
+ ​     limitations under the License. 
+ ​--> 
+ ​<​resources​ ​xmlns​:​xliff​=​"​urn:oasis:names:tc:xliff:document:1.2​"​> 
+ ​    ​<!--​ 自定义搜索栏 ​--> 
+ ​    <​string​ ​name​=​"​custom_seekbar_value​"​>自定义设置: <​xliff​:​g​ ​id​=​"​v​"​>%s</​xliff​:​g​></​string​> 
+ ​    <​string​ ​name​=​"​custom_seekbar_default_value​"​>默认设置</​string​> 
+ ​    <​string​ ​name​=​"​custom_seekbar_default_value_to_set​"​>修改默认设置: <​xliff​:​g​ ​id​=​"​v​"​>%s</​xliff​:​g​>\长按检测</​string​>
```
Comment (urain39) [inline parts/res/values-cn/strings.xml:21]: “\n”不需要翻译，这里是回车符号。后面半句是“长按以设置的意思”。

**Review by urain39 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**